tests/base.py

```diff
@@ -239,5 +242,17 @@ def __init__(self):
         self.template_system = self
         self.name = 'mako'
 
+    def _activate_plugins_of_category(self, category):
```

Comment:
Doesn't seem to work. What about moving the method to `nikola.utils` and calling it explicitly with a `site` instance?